changelog.d/507.bugfix

```diff
@@ -0,0 +1 @@
+Add a configuration widget for Jira.
```

Comment:
@Half-Shot This is the same [changelog entry](https://github.com/matrix-org/matrix-hookshot/pull/502/files#diff-76d69f49eb80d1d55e43a13b13cc35d753c6fc68569942622ee07b64b53e16d9R1) as #502's, but with a different filename; is that what you wanted?

Reply:
This is good.